stratum/hal/lib/common/openconfig_converter.cc

```diff
@@ -246,9 +246,8 @@ SingletonPortToInterfaces(const SingletonPort &in) {
   interface_key.set_name(in.name());
   auto interface = interface_key.mutable_interface();
 
-  // SingletonPort.id -> /interfaces/interface/state/id(ifindex)
+  // SingletonPort.id -> /interfaces/interface/config/id
   interface->mutable_id()->set_value(in.id());
-  interface->mutable_ifindex()->set_value(in.id());
```

Comment:
@pudelkoM -- see we no longer translate include the ifindex in either direction of translation.